.changeset/strict-lamps-heal.md

```
@@ -0,0 +1,11 @@
+---
+'@mastra/core': minor
+---
+
+Added minimal persistence for resuming agent tool approvals without storing complete workflow snapshots.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)